example.axiosConfig.js

```diff
@@ -4,6 +4,6 @@ const instance = axios.create({
   baseURL: 'https://app-hrsei-api.herokuapp.com/api/fec2/rfp',
 });
 
-instance.defaults.headers.common['Authorization'] = '**YOUR_AUTH_TOKEN**';
+instance.defaults.headers.common.Authorization = '**YOUR_AUTH_TOKEN**';
```

Comment:
Did this also work? The linter was mad, but I just kept it as it was on axios. Anyway, everything looks great!